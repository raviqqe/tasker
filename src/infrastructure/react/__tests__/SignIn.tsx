import React from "react";
import { create } from "react-test-renderer";
import { SignIn } from "../SignIn";

it("renders", () => {
  expect(
    create(<SignIn signIn={() => undefined} />).toJSON()
  ).toMatchSnapshot();
});
