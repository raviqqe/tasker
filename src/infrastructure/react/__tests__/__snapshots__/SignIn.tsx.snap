// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<button
  className="sc-bdfBQB sc-gsTEea eTilcL huREHC"
  onClick={[Function]}
>
  Sign in
</button>
`;
